sdk/scheduler/src/main/java/com/mesosphere/sdk/state/DefaultSchemaVersionStore.java

```diff
@@ -1,98 +0,0 @@
-package com.mesosphere.sdk.state;
```

Comment:
Reviewer note: this was moved to `SchemaVersionStore.java` without further changes